api/app/Generators/CandidateProfileCsv.php

```diff
@@ -218,15 +233,26 @@ public function generate()
     private function generatePoolHeaders()
     {
 
-        Pool::with(['generalQuestions', 'poolSkills' => ['skill']])
+        Pool::with(['generalQuestions', 'screeningQuestions', 'poolSkills' => ['skill']])
             ->whereHas('poolCandidates', function ($query) {
                 $query->whereIn('id', $this->ids);
             })->chunk(100, function ($pools) {
                 foreach ($pools as $pool) {
                     if ($pool->generalQuestions->count() > 0) {
                         foreach ($pool->generalQuestions as $question) {
-                            $this->questionIds[] = $question->id;
-                            $this->generatedHeaders['general_questions'][] = $question->question[$this->lang];
+                            $this->generalQuestionIds[] = $question->id;
+                            $this->generatedHeaders['general_questions'][] = $this->lang == 'en' ?
+                            'General question: '.$question->question[$this->lang] :
+                            'Question générale: '.$question->question[$this->lang];
+                        }
+                    }
+
+                    if ($pool->screeningQuestions->count() > 0) {
+                        foreach ($pool->screeningQuestions as $question) {
+                            $this->screeningQuestionIds[] = $question->id;
+                            $this->generatedHeaders['screening_questions'][] = $this->lang == 'en' ?
+                            'Screening question: '.$question->question[$this->lang] :
+                            'Question de sélection: '.$question->question[$this->lang];
                         }
```

Comment:
https://github.com/GCTC-NTGC/gc-digital-talent/blob/e0381a2a9f6b7dff87e85b5151ed7167ef8e8d27/api/lang/fr/headings.php#L60

![bekphnqftcb41](https://github.com/user-attachments/assets/8f661add-40c3-47ad-85cd-6da3fe55a5a6)

Reply:
"Singular verse plural", remarked the 🟢 👺